Source/WebCore/rendering/cocoa/RenderThemeCocoa.mm

```diff
@@ -555,8 +647,10 @@ static inline bool canShowCapsLockIndicator()
     if (mediaElement.document().settings().hostedBlurMaterialInMediaControlsEnabled())
         mediaControlsStyleSheets.append(glassMaterialMediaControlsStyleSheet());
 
-    if (mediaElement.document().settings().mediaControlsMacInlineSizeSpecsEnabled())
+    if (mediaElement.document().settings().mediaControlsMacInlineSizeSpecsEnabled()) {
```

Comment:
This is OK as-is, although `mediaControlsMacInlineSizeSpecsEnabled` is not slightly misnamed.